delivery/services/organise_service.py

```diff
@@ -128,27 +126,35 @@ def organise_project(self, runfolder, project, organised_projects_path, lanes):
 
         return organised_project
 
-    def organise_project_file(self, project_file, organised_project_path, project_file_base=None):
+    def organise_project_file(self, project_file, organised_project_path):
         """
         Find and symlink the project report to the organised project directory.
 
         :param project: a Project instance representing the project before organisation
         :param organised_project: a Project instance representing the project after organisation
         """
-        project_file_base = project_file_base or self.file_system_service.dirname(project_file.file_path)
+
+        # the relative path from the project file base to the project file (e.g. plots/filename.png)
+        relpath = self.file_system_service.relpath(
+            project_file.file_path,
+            project_file.base_path
+        )
 
         # the full path to the symlink
         link_name = os.path.join(
             organised_project_path,
-            self.file_system_service.relpath(
-                project_file.file_path,
-                project_file_base))
+            relpath
+        )
```

Comment:
yes, it can be other files now as well.

no, `base_path` and `organised_project_path` need not be the same

e.g. 
```
file_path = "/path/to/example/file_name"
base_path = "/path/to"
organised_project_path = "/path/to/organised/"
```
would mean that
```
link_name = "/path/to/organised/example/file_name"
```